torch/csrc/api/include/torch/nn/modules/activation.h

```diff
@@ -32,6 +32,29 @@ class TORCH_API ELUImpl : public torch::nn::Cloneable<ELUImpl> {
 
 TORCH_MODULE(ELU);
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ SELU ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+/// Applies the selu function element-wise.
+/// See https://pytorch.org/docs/master/nn.html#torch.nn.SELU to learn
+/// about the exact behavior of this module.
+class TORCH_API SELUImpl : public torch::nn::Cloneable<SELUImpl> {
+ public:
+  SELUImpl() : SELUImpl(SELUOptions()) {}
+  explicit SELUImpl(const SELUOptions& options_);
```

Comment:
Would we be able to combine the above two constructors and just have the following?
```cpp
  explicit SELUImpl(const SELUOptions& options_ = {});
```

Reply:
Yup, we could do that! I was just keeping consistent with the rest of the activation constructors. Would you like me to update it?

Reply:
Yup updating it would be awesome, thanks a lot for the great work @jon-tow ! :D